examples/medplum-demo-bots/src/deduplication/README.md

```diff
@@ -0,0 +1,95 @@
+# Patient Deduplication Demo
+
+This folder contains a reference implementation for a simple, human-in-the-loop patient deduplication pipeline, based on the guidance in [Patient Deduplication Architectures](https://www.medplum.com/docs/fhir-datastore/patient-deduplication#architecture-overview).
+
+The implementation consists of two bots:
+
+- [`find-matching-patients`](./find-matching-patients.ts) - Identifies duplicated Patients and performs a series of steps:
+  - Check for the first name, last name, birth date, and zip code.
+  - If it's a match, it will check a list to see if it's not on the doNotMatch list
+  - Creates a `RiskAssessment` representing the candidate match
+  - Creates a `Task` for a human to review the match
+- [`merge-matching-patients`](./merge-matching-patients.ts) -The other that merges the two Patients into one
+  - [Link the Patient records](https://www.medplum.com/docs/fhir-datastore/patient-deduplication#linking-patient-records-in-fhir)
+  - [Merge the Contact info](https://www.medplum.com/docs/fhir-datastore/patient-deduplication#merge-rules)
+  - [Rewrite Clinical Resource](https://www.medplum.com/docs/fhir-datastore/patient-deduplication#rewriting-references-from-clinical-data)
+  - Delete the source record, if the user has requested it
+
+This implementation also consists of FHIR bundles with sample data, which can be uploaded using the [batch upload tool](https://www.medplum.com/docs/tutorials/importing-sample-data):
+
+- [patient-data.json](./patient-data.json) - Three sample `Patient` resources, two of whom are known duplicates, and one of which is a false positive
+- [merge-questionnaire.json](./merge-questionnaire.json) - An example `Questionnaire` resource that describes a form that a reviewer might use to evaluate a candidate match.
+
+## Setup
+
+To run and deploy your Bot do the following steps:
+
+Install:
```

Comment:
This does seem repetitive to the base bot repo, which is not ideal for docs, but for readme may be preferable for some reason. Not sure the right editorial policy here to maintain SSOT.

Reply:
Discussed offline. Keeping this in for easy of linking